lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if type(text) != str:
+        return None
+    f = filter(str.isalpha, text)
```

Comment:
can you make it in one line?

Reply:
i don't think i can...... can you explain? sorry, i forgot to ask you about it...

Reply:
I absolutelly sure that you can write filter + join + lower + casting to list